frontend/tests/README.md

```diff
@@ -129,3 +128,103 @@ All the E2E test commands can be found in `frontend/Makefile` and `frontend/pack
       1. For more information, visit [Playwright Inspector](https://playwright.dev/docs/debug)
 
       ![Playwright Inspector](https://user-images.githubusercontent.com/6309723/196775284-81bd4853-b4c6-45c4-827b-22e97b97ee06.png)
+
+---
+
+# Test/Env Differential
+
+Where tests are skipped vs. run in different environments.
```

Comment:
I have a [PR](https://github.com/chanzuckerberg/single-cell-data-portal/pull/3516) that annotates e2e tests that run in different envs, not sure if it's worth mentioning that here?